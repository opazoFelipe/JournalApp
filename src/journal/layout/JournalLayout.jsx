import { Box } from "@mui/system"

const drawerWidth = 240

export const JournalLayout = ({ children }) => {
  return (
    <Box sx={{ display: 'flex' }}>
        {/* NavBar drawerWidth */}

        {/* SideBar drawerWidth */}

        <Box 
            component='main'
            sx={{ flexGrow: 1, p: 3}}
        >
            {/* Toolbar */}

            { children }

        </Box>

    </Box>
  )
}
