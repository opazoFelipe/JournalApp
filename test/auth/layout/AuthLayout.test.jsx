import { render, screen } from "@testing-library/react"
import { AuthLayout } from "../../../src/auth/layout/AuthLayout"

describe('Pruebas en <AuthLayout />', () => {

    test('debe hacer match con el snapshot', () => {

        const { container } = render(<AuthLayout title={''} />)

        expect(container).toMatchSnapshot()
    })

})