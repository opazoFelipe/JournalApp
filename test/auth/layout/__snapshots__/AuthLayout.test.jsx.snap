// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AuthLayout /> debe hacer match con el snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column css-ro1yt6-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-3 box-shadow css-1ar3nvi-MuiGrid-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-1jcm1rz-MuiTypography-root"
      />
    </div>
  </div>
</div>
`;
